recipes/barrnap/0.3/build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-mkdir -p $PREFIX
-mv ./* $PREFIX/
+mkdir -p $PREFIX/bin
+mv ./* $PREFIX/bin
```

Comment:
The big advantage is that the tool is fixed. $PREFIX/bin should only contain executables. If this is not ensured, it could e.g. be that another tool overrides some of the files you copy to $PREFIX/bin. But if barrnap relies on these, you get unwanted side effects (e.g. barrnap not working any more) as well. Ideally, we do a wrapper as a quick fix, and report the problems to upstream, in the hope that the devs are dedicated enough to fix their package.

In a way, we follow good practice for unix packaging. You won't find any debian package that pollutes /usr/bin with unwanted files as well.